### ISSUE TITLE
Runtime warning while navigating to ```menu```

### ISSUE THREAD

**Issue by cheng-alvin** [Chillandchat/mobile#98]
Hey there!
This is a bug report regarding warning messages found when navigating the ```menu```. As the logs shown below:

### Logs
``` 
WARN  Found screens with the same name nested inside one another. Check:

menu, menu > menu

This can cause confusing behavior during navigation. Consider using unique names for each screen instead. 
    at BaseNavigationContainer (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:101832:28)
    at ThemeProvider (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:106907:21)
    at NavigationContainerInner (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:106799:26)
    at RCTView
    at View (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48291:43)
    at Provider (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:110444:21)
    at anonymous (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:101051:52)
    at withDevTools(Anonymous) (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:100707:27)
    at RCTView
    at View (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48291:43)
    at RCTView
    at View (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48291:43)
    at AppContainer (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48168:36)
    at main(RootComponent) (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:84659:28)
``` 

### Reproduction steps
- Start and enter the development server.
- Load into the ```login``` screen.
- Enter username and password in text fields respectively.
- Tap "login".

**[Bug occurs here]**

Please resolve this issue soon.
Thank you!

**Comment by cheng-alvin** [comment]
> Hey there! This is a bug report regarding warning messages found when navigating the `menu`. As the logs shown below:
> 
> ### Logs
> ```
> WARN  Found screens with the same name nested inside one another. Check:
> 
> menu, menu > menu
> 
> This can cause confusing behavior during navigation. Consider using unique names for each screen instead. 
>     at BaseNavigationContainer (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:101832:28)
>     at ThemeProvider (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:106907:21)
>     at NavigationContainerInner (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:106799:26)
>     at RCTView
>     at View (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48291:43)
>     at Provider (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:110444:21)
>     at anonymous (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:101051:52)
>     at withDevTools(Anonymous) (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:100707:27)
>     at RCTView
>     at View (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48291:43)
>     at RCTView
>     at View (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48291:43)
>     at AppContainer (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:48168:36)
>     at main(RootComponent) (http://192.168.0.244:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:84659:28)
> ```
> 
> ### Reproduction steps
> * Start and enter the development server.
> * Load into the `login` screen.
> * Enter username and password in text fields respectively.
> * Tap "login".
> 
> **[Bug occurs here]**
> 
> Please resolve this issue soon. Thank you!

Note that this bug or warning does not affect the runtime behavior by any means, and is suspected to be problematic or confusing in the future.

**Comment by cheng-alvin** [comment]
After further analysis, due to the fact that we have a navbar with within another screen, I have found that this warning is being thrown due to the fact that there is a ```menu``` screen nested inside the ```menu``` screen which points to the ```ControlCenter``` navigation group. All we need to do is to rename a navigation screen.

**Comment by cheng-alvin** [comment]
This issue is now being fixed in #99 and will be continued there.
